### PR TITLE
feat: Update "/launch" page

### PR DESCRIPTION
**Pull request by mohandast52**
- Preview [here](https://olas-website-git-mohan-launch-updates-autonolas.vercel.app/launch)

### CODE REVIEW

File: common-util/classes.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const SUB_HEADER_CLASS = 'tracking-tight font-extrabold text-3xl md:font-bold lg:text-4xl';
+
+export const P_XL_CLASS = 'text-2xl';
+
+export const P_LARGE_CLASS = 'text-2xl';
```
Comment (Tanya-atatakai) [inline common-util/classes.js:5]: should `P_XL_CLASS` and `P_LARGE_CLASS` values be the same?

File: common-util/classes.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export const SUB_HEADER_CLASS = 'tracking-tight font-extrabold text-3xl md:font-bold lg:text-4xl';
+
+export const P_XL_CLASS = 'text-2xl';
```
Comment (Tanya-atatakai) [inline common-util/classes.js:3]: maybe just `TEXT_XL_CLASS`? can be applied to spans as well, not just to p tag

File: common-util/classes.js (hunk 1 above)
Comment (Tanya-atatakai) [reply common-util/classes.js:5]: actually, don't see where `P_XL_CLASS` is used, maybe can reove

**Review by Tanya-atatakai — APPROVED** [review]
lgtm!

File: components/LaunchPage/CreateAnAutonomousAiAgent.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    Involves developing a simple extension contract that embeds these KPIs in
+    code.
+    {' '}
+    <Link
+      href="https://github.com/valory-xyz/autonolas-staking-programmes/blob/main/contracts/mech_usage/MechActivityChecker.sol"
+      className="text-purple-600"
+      target="_blank"
+    >
+      Example here
+    </Link>
```
Comment (truemiller) [inline components/LaunchPage/CreateAnAutonomousAiAgent.jsx:19]:
```suggestion
    </Link>.
```

File: components/LaunchPage/CreateAnAutonomousAiAgent.jsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          </li>
+        ))}
+      </ol>
+
+      <p className="mt-2">
+        More on staking
+        {' '}
+        <Link href="https://staking.olas.network/" className="text-purple-600">
+          here
+        </Link>
```
Comment (truemiller) [inline components/LaunchPage/CreateAnAutonomousAiAgent.jsx:57]:
```suggestion
        </Link>.
```

**Review by Tanya-atatakai — APPROVED** [review]

File: components/LaunchPage/CreateAnAutonomousAiAgent.jsx (hunk 4 above)
Comment (truemiller) [reply components/LaunchPage/CreateAnAutonomousAiAgent.jsx:70]: it's not really a matter of design/style, but rather grammar. the sentence is left hanging without an ending full-stop/period.

not really a blocker however, will approve.

**Review by truemiller — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**